README.md

```diff
@@ -6,6 +6,97 @@ channel branches (e.g. `channels/8/edge`).
+ - Corretto does not provide `glib` libraries imported by the URL proxy selector. `java.net.ProxySelector.getDefault()` call will always return `Direct Proxy`.
+
+The JRE differences itself are minimal. Th chiselled image removes libawt_xawt.so and libsplashscren.so along with accessibility support. Executables, except `java`, are removed from `jre/bin`.
+Note: chiselled docker at the moment does not provide classes.jsa (Class Data Cache) in line with Temurin JRE and it has to be generated.
+
+Below are image sizes of the deployed `acmeair` benchmark application
+|Image|Base Image|Uncompressed Size| Compressed Size|
+|-----|---|----| ----------------------------|
+| acmeair deployed to WebSphere Liberty official image| websphere-liberty:23.0.0.1-full-java8-ibmjava| 772MB| 507MB|
+| acmeair and WebSphere Liberty deployed to chiselled-jre| ubuntu/chiselled-jre:8_edge | 631MB | 407MB |
+| acmeair and WebSphere Liberty deployed to Corretto| 8u352-alpine3.14-jre | 618MB | 402MB |
+| acmeair deployed to Apache Tomcat | tomcat:9.0.71-jre8-temurin-jammy | 258MB | 108MB |
+| acmeair and Apache Tomcat deployed to chiselled-jre | ubuntu/chiselled-jre:8_edge | 164MB | 76M |
+| acmeair and Apache Tomcat deployed to Corretto| 8u352-alpine3.14-jre | 151MB | 71M |
+| acmeair Stanalone Spring Boot on Temurin | eclipse-temurin:8u352-b08-jre-jammy | 244MB | 99MB |
+| acmeair Stanalone Spring Boot | ubuntu/chiselled-jre:8_edge | 146MB | 65MB|
+| acmeair Stanalone Spring Boot on Corretto | 8u352-alpine3.14-jre | 133MB | 61MB|
+
+### Startup time
+
+The startup times were evaluated by starting a Spring Boot standalone container repeatedly and measuring the total JVM time until the start of the application as per Spring Boot logs.
+The table below shows average startup times (seconds):
+|chiselled-jre| chiselled-jre with Class Data Caching| Temurin | Temurin with Class Data Caching| Corretto | Corretto with Class Data Caching |
+|-----|---|----| -------------|---| ---|
+|1.03 |0.99328125| 1.04615625|0.989625| 1.1124375 |  1.06978125 |
+
+Adding CDS (Class Data Sharing) to `acmeair` allows a 5% startup improvement at expense of a 6MB compressed (and 20MB uncompressed) image size increase.
+
+### Throughput tests
+
+The throughput tests were performed using Apache JMeter 5.5 on `acmeair` application with the following command: ``./apache-jmeter-5.5/bin/jmeter -n -t AcmeAir-v5.jmx -DusePureIDs=true -JHOST=localhost -JPORT=9080 -j performance.log -JTHREAD=1 -JUSER=9999 -JDURATION=120 -JRAMP=60 ;``
+
+The table below shows average requests per second:
+
+| WebSphere Liberty | WebSphere Liberty (chiselled) | Apache Tomcat | Apache Tomcat (chiselled) | Standalone (Temurin) | Standalone (chiselled) |  Standalone (corretto) |
+|-------------------|-------------------------------|---------------|-------------|---------|--------| ---|
+| 1553.7 | 1704.5 | 1687.7 | 1733.2 | 1778 | 1773.6 | 1798 |
```

Comment:
So, Standalone is more or less consistent; but vs Websphere and Tomcat, Chiselled is about 10% better?
Do I read that correctly? (just checksum-ing my understanding, I agree with your conclusion that performance is more or less the same)